tendrl/node_agent/node_sync/sds_detect.py

```diff
@@ -106,34 +106,36 @@ def sync(sync_ttl, node_status_ttl):
                             if dc.detected_cluster_id and \
                                 dc.detected_cluster_id != sds_details.get(
                                     'detected_cluster_id'):
-
                                 # Gluster peer list has changed
                                 integration_id = \
                                     NS.tendrl_context.integration_id
                                 etcd_utils.write(
                                     integration_index_key,
                                     integration_id
                                 )
-                                # Set the cluster status as new peer detected
                                 _cluster = NS.tendrl.objects.Cluster(
                                     integration_id=integration_id
                                 ).load()
-                                _cluster.status = "new_peers_detected"
-                                _cluster.save()
-                                # Raise an alert regarding the same
-                                msg = "New peers identified in cluster: %s. " \
```

Comment:
So you want to say only if cluster managed then cluster expansion required? makes sense..

Reply:
yes, let's take an example if we have two nodes detected. if user peer probe another node before managing a cluster then it will display two clusters as detected. Because multiple detected cluster_id will present in ETCD.